tests/tests.cpp

```diff
@@ -38,7 +38,7 @@ class EngineBasicTest : public testing::Test {
     random_str(&str_pool[0], str_pool_length);
     configs.pmem_file_size = (16ULL << 30);
     configs.populate_pmem_space = false;
-    configs.hash_bucket_num = (1 << 5);
+    configs.hash_bucket_num = (1 << 10);
     configs.hash_bucket_size = 64;
```

Comment:
What's this two configs?
We should let user decide hash entries per bucket and hash buckets per slot, but not the hash_bucket_size

Reply:
agree

Reply:
but use size, it's clear to align bucket size to cache line